crates/ruff/src/rules/flake8_2020/rules.rs

```diff
@@ -175,21 +175,23 @@ pub fn subscript(checker: &mut Checker, value: &Expr, slice: &Expr) {
 /// YTT103, YTT201, YTT203, YTT204, YTT302
 pub fn compare(checker: &mut Checker, left: &Expr, ops: &[Cmpop], comparators: &[Expr]) {
     match &left.node {
-        ExprKind::Subscript { value, slice, .. } if is_sys(checker, value, "version_info") => {
-            if let ExprKind::Constant {
+        ExprKind::Subscript(ast::ExprSubscript { value, slice, .. })
+            if is_sys(checker, value, "version_info") =>
+        {
+            if let ExprKind::Constant(ast::ExprConstant {
                 value: Constant::Int(i),
                 ..
-            } = &slice.node
+            }) = &slice.node
             {
                 if *i == BigInt::from(0) {
                     if let (
                         [Cmpop::Eq | Cmpop::NotEq],
-                        [Located {
+                        [Attributed {
```

Comment:
I personally find `Attributed` somewhat more difficult to understand than `Located` because it's very generic. Attributed with what. But I don't feel strongly about it. 

A term that is used by many other compilers for a wrapper that adds source locations is [`Spanned`](https://www.google.com/search?client=firefox-b-d&q=rust+spanned) or (since we don't have a span type), could also name it `Ranged`. But again, I'm okay with keeping it. Just my two cents ;)

Note: I'm exploring how we can have location information on every node. We'll probably need this to correctly associate comments with nodes.

Reply:
`Ranged` sounds good. Do you want it on this PR?

Reply:
We can do it in another PR to avoid unnecessary rebasing